corehq/apps/ota/views.py

```diff
@@ -64,10 +61,8 @@ def search(request, domain):
     for key, value in criteria.items():
         search_es = search_es.case_property_query(key, value, fuzzy=(key in fuzzies))
     results = search_es.values()
-    if should_use_sql_backend(domain):
-        cases = [get_instance_from_data(CommCareCaseSQLSerializer, result) for result in results]
-    else:
-        cases = [CommCareCase.wrap(flatten_result(result)) for result in results]
+    # Even if it's a SQL domain, we just need to render the results as cases, so CommCareCase.wrap will be fine
+    cases = [CommCareCase.wrap(flatten_result(result)) for result in results]
```

Comment:
Hmm... I didn't think about attachments. These currently aren't added to the case search fixture in either case (couch or sql). I'm not sure we need to send down attachments with search results though, but we should probably make explicit somewhere that these aren't included in the cases sent down in the search fixture.